src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="app">
+      <div class="app__wrapper">
+        <form
+          class="search-bar"
+          data-qa="big"
+        >
+          <label class="search-input">
```

Comment:
```suggestion
          <label class="search-label">
```
it's suits better

Reply:
In this case I do not use label separately from input, input is a part of label , like a solid BEM block label>input